bin/spectatord_main.cc

```diff
@@ -51,6 +51,8 @@ ABSL_FLAG(PortNumber, statsd_port, PortNumber(8125),
 ABSL_FLAG(bool, enable_socket, true, "Enable UNIX domain socket support.");
 ABSL_FLAG(std::string, socket_path, "/run/spectatord/spectatord.unix",
           "Path to the UNIX domain socket.");
+ABSL_FLAG(std::string, agg_uri, "",
```

Comment:
Ideally the option name would match the name used for the config class (`uri`) to avoid confusion.